integrations/spring/opendal-spring-boot-starter/src/test/java/org/apache/opendal/spring/config/OpenDALAutoConfigurationTest.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.opendal.spring.config;
+
+import org.apache.opendal.spring.OpenDALApplication;
+import org.apache.opendal.spring.core.OpenDALOperations;
+import org.apache.opendal.spring.core.OpenDALProperties;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit.jupiter.SpringJUnitConfig;
+
+@SpringJUnitConfig
+@SpringBootTest(classes = OpenDALApplication.class)
+public class OpenDALAutoConfigurationTest {
+    @Autowired
+    private OpenDALProperties openDALProperties;
+
+    @Autowired
+    private OpenDALOperations openDAL;
+
+    @Test
+    public void properties_bean_should_be_declared() {
+        Assertions.assertNotNull(openDALProperties);
+    }
+
+    @Test
+    public void bean_should_be_declared() {
+        Assertions.assertNotNull(openDAL);
```

Comment:
`OpenDALProperties` is a tagged interface, assert the real type here?

Reply:
fixed